vue/mixins/logic.js

```diff
@@ -68,6 +74,23 @@ export const logicMixin = {
         this.$bus.bind("refresh", this.$forceUpdate);
     },
     methods: {
+        validInitials(initials) {

```

Comment:
initialsWithinRange

Reply:
Comment making sure talk about the fact that the max and min values are naive approximations